### PR TITLE
Add JS script for shopping list

### PR DESCRIPTION
**Pull request by adnacg**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 3

Completeness Level (1-5): 5

What did you think of this deliverable?: Difficult to understand how to access the key/value

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
+// 1.) Go through the items and find all results that have kind of shopping#product. Print the count of these results. Where else is this count information stored in the search results?
```
Comment (scott88lee) [inline script.js]: wow....

Comment (scott88lee) [reply script.js]: Sorry, I left it a wow and didn't elaborate. You're the only one that gave many solutions to a problem. I'm impressed. 

Along the same line, I'll recommend you look into another way of looping.
See if you can do it with a forEach method.

Comment (adnacg) [reply script.js]: not sure I understand, isn't my options 1 already using a `.forEach()`?

Comment (scott88lee) [reply script.js]: Awesome, sorry didn't really read very in depth

**Pull request closed without merging**